app/models/prison.rb

```diff
@@ -1,32 +1,62 @@
 class Prison
+  attr_accessor :address, :adult_age, :booking_window, :canned_responses,
+    :email, :enabled, :finder_slug, :lead_days,
+    :name, :nomis_id, :phone, :reason,
+    :slot_anomalies, :slots, :unbookable,
+    :works_weekends
+
   DEFAULT_LEAD_DAYS = 3.freeze
   DEFAULT_BOOKING_WINDOW = 28.freeze
```

Comment:
Think I'm probably to blame for this.. :cry: total dufus. Integers & Floats are frozen by default.